client/src/App.js

```diff
@@ -1,17 +1,19 @@
 import React, { Component } from 'react'
-import Home from './components/Home/Home'
+import Landing from './components/Landing/Landing'
 import QuizPage from './components/QuizPage/QuizPage'
 import Result from './components/Result/Result'
+import RocketList from './components/RocketList/RocketList'
 import { Route, withRouter } from 'react-router-dom'
 import './App.css'
 
 class App extends Component {
   render () {
     return (
-      <div>
-        <Route exact path='/' component={Home} />
+      <div className='app'>
+        <Route exact path='/' component={Landing} />
         <Route exact path='/quiz' component={QuizPage} />
         <Route exact path='/result' component={Result} />
+        {localStorage.getItem('jwt') && localStorage.getItem('id') ? <Route exact path='/rocketlist/:userId' component={RocketList} /> : null} 
```

Comment:
There's actually a space at the end of this line which is triggering ESLint to yell on my local copy, can you remove it?

Reply:
There was an extra white space on line 16. I removed it.